management/signing_key_test.go

```diff
@@ -2,70 +2,56 @@ package management
 
 import (
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestSigningKey(t *testing.T) {
 	var kid string
 
-	// Our last test revokes the key used to sign the token we're currently
-	// using, so we need to re-authenticate so that subsequent tests still work.
-	t.Cleanup(func() {
-		initTestManagement()
-	})
-
 	t.Run("List", func(t *testing.T) {
-		ks, err := m.SigningKey.List()
-		if err != nil {
-			t.Fatal(err)
-		}
-		if len(ks) == 0 {
-			t.Error("expected at least one key to be returned")
-		}
+		signingKeys, err := m.SigningKey.List()
+		assert.NoError(t, err)
+		assert.NotEmpty(t, signingKeys, "expected at least one key to be returned")
 
 		// save kid for later use
-		kid = ks[0].GetKID()
-
-		t.Logf("%v\n", ks)
+		kid = signingKeys[0].GetKID()
```

Comment:
Could these tests run in any order?

Reply:
Good catch! Thanks for spotting this:) I somehow missed it. Fixed in https://github.com/auth0/go-auth0/pull/60/commits/9c910cdb0b3bf78e60a012783756a36cfb0985a1.